fuzz/README.md

```diff
@@ -0,0 +1,26 @@
+# fuzz-sbat
+
+See the [Rust Fuzz Book] for the initial setup.
+
+Run the fuzzer:
+```
+cargo +nightly fuzz run combo
+```
+
+Generate coverage:
+
+```
+rustup component add --toolchain nightly llvm-tools-preview
+
+cargo +nightly fuzz coverage combo
+
+cargo +nightly cov -- show target/x86_64-unknown-linux-gnu/coverage/x86_64-unknown-linux-gnu/release/combo \
```

Comment:
cargo cov doesn't work for me for some reason. It complains that `show` is not an option.

However, if I run llvm-cov instead of trying to use cargo cov it gives me something.

Reply:
Good point, `cargo cov` may just have been some tool I happened to have installed.

I've updated the instructions and switched to `grcov`, which is better maintained and has much nicer output.

Can you give the instructions another try and verify they work for you?

Reply:
Yes, these new instructions worked with the Makefile and grcov